### DISCUSSION TITLE
Module parse failed

### DISCUSSION THREAD

**Discussion by syafiqmuda** [tsparticles/vue2#24] [Q&A] [answered]
hello,

anyone received any error about:

You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders

I am using nuxt 2 framework.

**Comment by matteobruni** [comment]
There's already a working Nuxt2 sample in this repository here: https://github.com/tsparticles/vue2/tree/main/apps/nuxt2

Please check it and see if you can find the solution to your problem.

**Reply by syafiqmuda** [reply]
I am really new to the nuxt2 framework, what have I done is:
1) in the "vue2\apps\nuxt2" file
2) I `npm install --f`
3) then try `npm run dev`

it show me a lot of error, and one of it is:
"Module build failed"


Installation / Initial process
1) `npm init nuxt-app <project-name>`
2) inside the `<project-name>`
3) inside vscode console `npm install vue2-particles`
4) create plugins file "vue2-particles"
5) add plugins inside the "nuxt.config.js"
6) `npm install`
7) `npm run dev`

![image](https://github.com/tsparticles/vue2/assets/80864791/21697f17-592d-436e-94aa-85e86fce6c81)

**Reply by matteobruni** [reply]
You need to run `pnpm install` and `pnpm run build` in the root folder, mixing existing package managers is not recommended.

**Reply by syafiqmuda** [reply]
So it is impossible to run using `npm`? Because my project build using `npm` and in order to run the project, I need to use `npm run dev`.

**Reply by syafiqmuda** [reply]
btw I create totally new project, using `npm` to install nuxt2 and then install tsparticle.

![image](https://github.com/tsparticles/vue2/assets/80864791/5a883920-5d36-4880-8213-27df2f37d1a3)

![image](https://github.com/tsparticles/vue2/assets/80864791/f665c1d3-b93e-4b45-9488-93eefd07297e)

**Reply by matteobruni** [reply]
You can't use NPM in this repository because it's using PNPM, in your projects you can use the package manager you prefer.

**Reply by syafiqmuda** [reply]
@matteobruni yes sir,

but let me clarify again, I create a new project using my own package manager, totally a new project, a fresh ones. Then I try `npm i vue2-particles` inside my project, from there I stuck.

**Reply by matteobruni** [reply, marked as answer]
I understood that, but you tried running this repository or a part of it using NPM. This repository is maintained using PNPM so changing it is not recommended. If you want to run something on this repository follow the instructions I gave you.
These packages are not PNPM dependent, they can be installed with all package managers.
As I said in the first answer, the Nuxt2 sample here is working fine, please check your configuration and dependencies with the Nuxt2 sample in this repository.

I've tried cloning again this repository to see if there were some issues, since my local copy was working fine. There were some build issues, and I've fixed them (they are already pushed) since there were some unspecified dependencies issues.

<img width="561" alt="Screenshot 2023-09-15 alle 09 29 58" src="https://github.com/tsparticles/vue2/assets/176620/0d9ed9b4-6aea-4dee-a07f-23491d6d489b">

You can see the changes here: https://github.com/tsparticles/vue2/commit/bfab35b1e62a1b2712fd18b6a1f72426e7c66e52#diff-600f30a0831f54ce7c023d5c2be791e1bfeda2a77357738b6e4300d759c3ee6d

**Reply by syafiqmuda** [reply]
Tq for the clarification, I consider my question are solved. Thank you so much for your time.
